inc/user.class.php

```diff
@@ -4499,12 +4499,12 @@ static function getUniquePersonalToken() {
    /**
     * Get token of a user. If not exists generate it.
     *
-    * @param $ID user ID
+    * @param $ID no longer used
```

Comment:
Improvement can be done to remove the 1st argument while sending a message for deprecated use.